docs/pages/admin-guides/access-controls/access-monitoring.mdx

```diff
@@ -543,7 +543,7 @@ ORDER BY
 Run custom audit query using `tctl audit query exec` command:
 
 ```code
-$ tctl audit query exec 'select event_time,identity_user from cert_create order by event_date limit 1;'
+$ tctl audit query exec 'select event_time,identity_user from cert_create order by event_date limit 1;' --days 7
```

Comment:
If the motivation for this info is that it wasn't obvious that the default is 7, then I don't think this is the right way to address that.

How about we add some text that explains this after the command? I would say something like this:

> By default, `tctl audit query exec` will search over the past 7 days.
> You can specify an alternate timeframe with the `--days=N` flag.

Reply:
That's fair. Updated to reflect that.